code/Form/GridFieldAddClassesButton.php

```diff
@@ -178,7 +178,7 @@ public function getHTMLFragments($grid)
         if (!$buttonName) {
             // provide a default button name, can be changed by calling {@link setButtonName()} on this component
             $objectName = $singleton->i18n_singular_name();
-            $buttonName = _t('SilverStripe\\Forms\\GridField\\GridField.Add', 'Add {name}', ['name' => $objectName]);
+            $buttonName = _t(GridField::class . '.AddNew', 'Add new {name}', ['name' => $objectName]);
```

Comment:
Note unrelated change from hardcoded string to `GridField::class` here to clean up while I'm here.